frontend/src/components/VideoCardPage/styles.css

```diff
@@ -0,0 +1,12 @@
+.root {
+    overflow: hidden;
+}
+
+.card_container {
+    margin: auto !important;
+}
+
+
```

Comment:
EMPTY LINE @tmaciulis22 PLEASE RESPOND WHAT SHOULD WE DOOOO

Reply:
SHAME!!!!